### DISCUSSION TITLE
convert command not found

### DISCUSSION THREAD

**Discussion by TheGreatDarkLord** [ImageMagick/ImageMagick#1991] [General]
CentOs

installed ImageMagick

but i get 
-bash: convert: command not found

imageck or magick also not found

if: php convert then
Could not open input file: convert

please help

**Comment by fmw42** [comment]
If ImageMagick (convert or magick) is not in your $PATH environment, then you need to provide the full path to convert or magick in your command. Use convert for IM 6 and magick for IM 7.  What do you get from in a bash terminal window if you type either 

`which convert `

or 

`type convert
`

then replace convert with magick and try again
